tests/unit/Infrastructure/DashboardBundle/Jira/Repository/IssueRepositoryTest.php

```diff
@@ -109,6 +109,13 @@ public function test_repository_flow()
         $this->assertSame(4, $found);
     }
 
+    public function test_empty_file_should_not_throw_an_exception()
```

Comment:
:+1: for adding the test. 

Off-topic: The class name of the test had me confused for a second, this should be DevelopmentIssueRepositoryTest. Let's fix this some other time :)